src/controllers/version/downloads.rs

```diff
@@ -141,6 +149,34 @@ pub async fn download(
     }
 }
 
+#[derive(Debug)]
+// #[error("see https://blog.rust-lang.org/2023/10/27/crates-io-non-canonical-downloads.html")]
```

Comment:
I am not sure do we need to remove this line.

Reply:
oooops, yeah, that was from an earlier implementation attempt and I somehow forgot to remove it 😂

Reply:
I originally wanted to derive `thiserror::Error` for the struct, but since there is a blanket implementation or `AppError` for `Error` things didn't quite work as intended, so I had to remove the `thiserror` derive again.